### PR TITLE
SFR-1404: Fix back on webreader returning to referer

### PR DESCRIPTION
**Pull request by jackiequach**
[Jira Ticket](http://jira.nypl.org/browse/SFR-1404)

### This PR does the following:
- Resolves back button issue when user follows link to reader from external site

### Testing requirements & instructions: 
-

### CODE REVIEW

**Comment by mwbenowitz** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr314-0rpvka99e0bl0tujpt8r1dcrwgcprqzw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6227842d01c6d20b853fd628

File: src/__tests__/Link.test.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { getBackUrl } from "../util/LinkUtils";
```
Comment (hokei) [inline src/__tests__/Link.test.tsx:1]: I'd move this test folder to be under /util/__tests__

**Review by hokei — CHANGES_REQUESTED** [review]
Looks good!  Update the changelog with a fix and change the filename here as well.

**Comment by mwbenowitz** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr314-0rpvka99e0bl0tujpt8r1dcrwgcprqzw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6227842d01c6d20b853fd628

**Review by hokei — APPROVED** [review]

**Comment by hokei** [comment]
🎉

**Pull request merged**